folly/portability/Time.cpp

```diff
@@ -275,9 +275,11 @@ extern "C" int clock_gettime(clockid_t clock_id, struct timespec* tp) {
   }
 }
 #else
+#if !__linux__
```

Comment:
What's causing `FOLLY_HAVE_CLOCK_GETTIME` to not be defined? This shouldn't be getting reached if that is defined properly.